pythonforandroid/recipes/python2/__init__.py

```diff
@@ -170,5 +173,52 @@ def do_python_build(self, arch):
         # print('python2 build done, exiting for debug')
         # exit(1)
 
+    def create_python_bundle(self, dirn, arch):
+        info("Filling private directory")
+        if not exists(join(dirn, "lib")):
+            info("lib dir does not exist, making")
+            shprint(sh.cp, "-a",
+                    join("python-install", "lib"), dirn)
+        shprint(sh.mkdir, "-p",
+                join(dirn, "include", "python2.7"))
+
+        libpymodules_fn = join("libs", arch.arch, "libpymodules.so")
+        if exists(libpymodules_fn):
+            shprint(sh.mv, libpymodules_fn, dirn)
+        shprint(sh.cp,
+                join('python-install', 'include',
+                        'python2.7', 'pyconfig.h'),
+                join(dirn, 'include', 'python2.7/'))
+
+        info('Removing some unwanted files')
+        shprint(sh.rm, '-f', join(dirn, 'lib', 'libpython2.7.so'))
+        shprint(sh.rm, '-rf', join(dirn, 'lib', 'pkgconfig'))
+
+        libdir = join(dirn, 'lib', 'python2.7')
+        site_packages_dir = join(libdir, 'site-packages')
+        with current_directory(libdir):
+            removes = []
+            for dirname, root, filenames in walk("."):
+                for filename in filenames:
+                    for suffix in EXCLUDE_EXTS:
+                        if filename.endswith(suffix):
+                            removes.append(filename)
+            shprint(sh.rm, '-f', *removes)
```

Comment:
Out of scope since it was already in our code base.
But it hurts a little bit knowing this entire block could be a single line command using `find`:
```sh
find . -name '*.ext`' -o -name '*.ext2' -o -name '*.ext3' -delete
```
But yes that's not portable. However I'm wondering if there's not a way to do it in Python that would look nicer

Reply:
I'm thinking more and more about making some better python abstractions for this. Something for the near future, perhaps.

Reply:
There is fnmatch in the standard lib that can be used.

Reply:
Actually I realize `endswith` can take tuple :smile: 
So we could actually go one `for` less:
```python
if filename.endswith(EXCLUDE_EXTS):
```
But I think it's OK to tackle it outside this pull request, since we have enough to check